certbot-apache/tests/testdata/gentoo_apache/apache/conf.d/apache2

```diff
@@ -26,7 +26,7 @@
 #
 # The following two options provide the default virtual host for the HTTP and
 # HTTPS protocol. YOU NEED TO ENABLE AT LEAST ONE OF THEM, otherwise apache
-# will not listen for incomming connections on the approriate port.
+# will not listen for incoming connections on the appropriate port.
```

Comment:
This typo should not be fixed, because this file is copied verbatim from Apache.

Reply:
Done.